src/quo2/components/onboarding/small_option_card/view.cljs

```diff
@@ -48,19 +48,22 @@
      :source              image}]])
 
 (defn small-option-card
+  "Variants: `:main` or `:icon`"
   [{:keys [variant title subtitle image on-press]
     :or   {variant :main}}]
   (let [main-variant?  (= variant :main)
         card-component (if main-variant? main-variant icon-variant)]
-    [rn/touchable-highlight
-     {:accessibility-label :small-option-card
-      :style               style/touchable-overlay
-      :active-opacity      1
-      :underlay-color      colors/white-opa-5
-      :on-press            on-press}
-     [rn/view {:style (style/card-container main-variant?)}
+    [rn/view
```

Comment:
you can use `[:<>` for top level components that has no props

Reply:
AFAIK, they're not the equivalent.
The use case of a `React.fragment` (`:<>`) is pretty different to the use case of  `rn/view`.
I added a small example that could be applied to the `small-option-card`:

```clojure
(defn my-card-fragment []
  [:<> ;; <- fragment used as top level component
   [child-1]
   [child-2]])

(defn my-card-view []
  [rn/view ;; <- view used as top level component
   [child-1]
   [child-2]])

(defn my-coll-of-cards []
  [rn/view {:style {:flex    1,
                    ; row-gap is supported on newer version of RN, but works for this example
                    :row-gap 20}}
   [my-card-fragment]
   [my-card-fragment]])

;; This will render:

;; child-1
;; -space of 20-
;; child 2
;; -space of 20-
;; child-1
;; -space of 20-
;; child 2

(defn my-coll-of-cards []
  [rn/view {:style {:flex    1,
                    :row-gap 20}}
   [my-card-view]
   [my-card-view]])

;; This will render:

;; rn/view containing:
;;   -child-1
;;   -child 2
;; -space of 20-
;; rn/view containing:
;;   -child-1
;;   -child 2
```